website/docs/d/datasource_compute_instance.html.markdown

```diff
@@ -0,0 +1,187 @@
+---
+layout: "google"
+page_title: "Google: google_compute_instance"
+sidebar_current: "docs-google-datasource-compute-instance-x"
+description: |-
+  Get a VM instance within GCE.
```

Comment:
Could we expand this a little? I think it's a bit more descriptive to say something like "information about" or something, just to be a little more explicit.

Reply:
Done